Makefile

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
Do we need to utilize [kyma genertic makefile](https://github.com/kyma-project/kyma/tree/main/common/makefiles)? As we use is in eventing-controller makefile?

Reply:
I asked this question in our tech talk and there was no comment on this. Hence, I left the standard kubebuilder targets. Let's bring our makefile targets step-by-step with the implementation as we probably use k3d and github action stuff that may influence our targets.